nixos/modules/services/continuous-integration/buildkite-agents.nix

```
@@ -35,6 +35,12 @@ let
         type = lib.types.str;
       };
 
+      extraGroups = lib.mkOption {
+        default = [ "keys" ];
+        description = lib.mdDoc "The groups the buildkite-agent user should belong to";
```

```suggestion
        description = lib.mdDoc "Groups the user for this buildkite agent should belong to";
```

Slight rewording, and making it clearer there's multiple users used (`buildkite-agent-${name}`)

---

thank you so much for the improvement @flokli !